Classes/Command/AcceptanceTestsCommand.php

```diff
@@ -0,0 +1,199 @@
+            $targetPackagePath . '/Tests/Acceptance/Fixtures',
+            $targetPackagePath . '/Tests/Acceptance/Application',
+            $targetPackagePath . '/Tests/Acceptance/Support/Extension'
+        ]);
+
+        // Create public folder which is required for e.g. acceptance tests to work
+        $publicFolderPath = $targetPackagePath . '/Resources/Public';
+        if (!is_dir($publicFolderPath)) {
+            $createPublic = $this->io->confirm(
+                'Resource/Public is necessary e.g. for acceptance tests. Do you want to create it now?',
+                true
+            );
+
+            if ($createPublic) {
+                $this->filesystem->mkdir([$publicFolderPath]);
+                // Ensure the folder will be detected by git and committed
+                $this->filesystem->touch([$publicFolderPath . '/.gitkeep']);
+            }
+        }
+
+        $files = $finder->in($templatePath)->files();
+
+        foreach ($files as $file) {
+            $target = $targetPackagePath . 'Tests' . explode('AcceptanceTests', $file->getRealPath())[1];
+
+            if (!is_file($target)) {
+                $content = $file->getContents();
+                $this->substituteMarkersAndSave($content, $target);
+            } else {
+                $overwrite = $this->io->confirm('File exists ' . basename($target) . '. Do you want to overwrite it?');
+                if ($overwrite) {
+                    $content = $file->getContents();
+                    $this->substituteMarkersAndSave($content, $target);
+                }
+            }
+        }
+
+        return 0;
+    }
+
+    /**
+     * Extend/prepare composer.json of the extension
+     * for acceptance tests
+     *
+     * @throws \JsonException
+     */
+    protected function updateComposerFile(string $packagePath): bool
+    {
+        $composerFile = $packagePath . '/composer.json';
+        $composerJson = file_get_contents($composerFile);
+        $composer = json_decode($composerJson, true);
+        if (json_last_error() !== JSON_ERROR_NONE) {
+            throw new \JsonException('Could not parse ' . $composerFile);
+        }
+
+        $namespace = rtrim($this->getNamespace(), '\\');
+
+        $addToComposerFile = [
+            'require-dev' => [
+                'codeception/codeception' => '^4.1',
+                'codeception/module-asserts' => '^1.2',
+                'codeception/module-webdriver' => '^1.1',
+                'typo3/testing-framework' => '^6.16.2'
+            ],
+            'autoload-dev' => [
+                'psr-4' => [
+                    $namespace . '\\Tests\\' => 'Tests/'
+                ]
+            ],
+            'config' => [
+                'vendor-dir' => '.Build/vendor',
+                'bin-dir' => '.Build/bin',
+            ],
+            'extra' => [
+                'typo3/cms' => [
+                    'app-dir' => '.Build',
+                    'web-dir' => '.Build/Web',
```

Comment:
please use `public` here as `Web` dates back to TYPO3 Flow times and should not be used in TYPO3 world.